### PR TITLE
bump to ANR 1.2.0

### PR DESCRIPTION
**Pull request by felipemadero**
Closes https://github.com/ava-labs/avalanche-cli/issues/147 https://github.com/ava-labs/avalanche-cli/issues/175

### CODE REVIEW

File: pkg/subnet/local_test.go [hunk 1]
```diff
@@ -226,9 +226,9 @@ func getTestClientFunc() (client.Client, error) {
 	c.On("Health", mock.Anything).Return(fakeHealthResponse, nil).Twice()
 	// Afterwards, change the VmId so that TestDeployToLocal has the correct ID to check
 	alteredFakeResponse := proto.Clone(fakeHealthResponse).(*rpcpb.HealthResponse) // new(rpcpb.HealthResponse)
-	alteredFakeResponse.ClusterInfo.CustomVms["bchain2"].VmId = testVMID
-	alteredFakeResponse.ClusterInfo.CustomVms["bchain2"].VmName = testVMName
-	alteredFakeResponse.ClusterInfo.CustomVms["bchain1"].VmName = "bchain1"
+	alteredFakeResponse.ClusterInfo.CustomChains["bchain2"].VmId = testVMID
+	alteredFakeResponse.ClusterInfo.CustomChains["bchain2"].ChainName = testVMName
```
Comment (holisticode) [inline pkg/subnet/local_test.go:230]: We might also want to rename `testVMName` to something like `testChainName`

**Review by holisticode — APPROVED** [review]
LGTM, just a little nit

File: pkg/subnet/local_test.go (hunk 1 above)
Comment (felipemadero) [reply pkg/subnet/local_test.go:230]: probably can be called testSubnetName, as it is being used as argument to DeployToLocalNetwork

Comment (felipemadero) [reply pkg/subnet/local_test.go:230]: will change to testChainName

**Pull request merged**